oem/ibm/libpldmresponder/file_io.cpp

```diff
@@ -930,13 +930,61 @@ Response Handler::fileAckWithMetaData(const pldm_msg* request,
         return CmdHandler::ccOnlyResponse(request, PLDM_INVALID_FILE_TYPE);
     }
 
-    rc = handler->fileAckWithMetaData(fileMetaData1);
+    rc = handler->fileAckWithMetaData(fileMetaData1, fileMetaData2,
+                                      fileMetaData3, fileMetaData4);
     auto responsePtr = reinterpret_cast<pldm_msg*>(response.data());
     encode_file_ack_with_meta_data_resp(request->hdr.instance_id, rc,
                                         responsePtr);
     return response;
 }
 
+Response Handler::newFileAvailableWithMetaData(const pldm_msg* request,
+                                               size_t payloadLength)
+{
+    std::cerr << " Got a newFile available with metadata command" << std::endl;
```

Comment:
pls remove this also ? or if you want to keep make it as std::cout as its not an error